gsa/src/web/pages/scanconfigs/editdialog.js

```diff
@@ -464,9 +467,13 @@ class NvtFamilies extends React.Component {
               <TableData>
                 {_('Total: {{count}}', {count: config.families.count})}
               </TableData>
-              <TableData align="end">
+              <TableData align="start">
                 {_('{{known}} of {{max}}', config.nvts)}
               </TableData>
+              {/* add empty cells to spread row to end of table */}
+              <TableData/>
+              <TableData/>
+              <TableData/>
```

Comment:
Seems strange to me. Shouldn't we remove the columns at all if we don't have content for them?